test/IDE/complete_function_builder.swift

```diff
@@ -93,3 +94,21 @@ struct EnumToVoidBuilder {
   static func buildBlock(_ :MyEnum, _: MyEnum, _: MyEnum) {}
 }
 func acceptBuilder(@EnumToVoidBuilder body: () -> Void) {}
+
+@_functionBuilder
+struct AnyBuilder {
+  static func buildBlock(_ components: Any...) -> Any { 5 }
+
+  static func #^IN_FUNCTION_BUILDER_DECL^#
+}
+
+// IN_FUNCTION_BUILDER_DECL: Begin completions, 8 items
+// IN_FUNCTION_BUILDER_DECL: Pattern/CurrNominal:                buildBlock(_ components: Any...) -> Any { {|}; name=buildBlock(_ components: Any...) -> Any {; comment=Required by every
```

Comment:
Hmm, I think inserting this would conflict with existing `buildBlock` declaration. Perhaps we should only suggest adding `build*` methods that aren't explicitly declared by the user in the function builder type?

Reply:
I had thought about that, but there are a number of cases where you might want to do some overloading here, and having a redundant completion is much better than having no completion.

Reply:
Hmm, sure, that seems reasonable. What do you think about using placeholders for types if a declaration already exists? Right now,  I think inserting one would trigger a redeclaration error and using placeholders for types can help avoid that.

Reply:
I don't feel like that level of tweaking is really worthwhile. Maybe we help in a few narrow places, and maybe it's just weird because two completions in a row produce different results.